keps/sig-node/127-user-namespaces/README.md

```diff
@@ -363,88 +549,463 @@ Note this section is a WIP yet.
+
+###### Are there any tests for feature enablement/disablement?
+
+We will add.
+
+We will test for when the field pod.spec.HostUsers is set to true, false
+and not set. All of this with and without the feature gate enabled.
 
-* **Are there any tests for feature enablement/disablement?**
+We will also unit test that, if pods were created with the new field
+pod.specHostUsers, then if the featuregate is disabled all works as expected (no
+user namespace is used).
+
+<!--
+The e2e framework does not currently support enabling or disabling feature
+gates. However, unit tests in each component dealing with managing data, created
+with and without the feature, are necessary. At the very least, think about
+conversion tests if API types are being modified.
+
+Additionally, for features that are introducing a new API field, unit tests that
+are exercising the `switch` of feature gate itself (what happens if I disable a
+feature gate after having objects written with the new field) are also critical.
+You can take a look at one potential example of such test in:
+https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
+-->
 
 ### Rollout, Upgrade and Rollback Planning
 
-Will be added before transition to beta.
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-* **How can a rollout fail? Can it impact already running workloads?**
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
 
-* **What specific metrics should inform a rollback?**
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
 
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+###### What specific metrics should inform a rollback?
 
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs,
-fields of API types, flags, etc.?**
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
 
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
 
 ### Monitoring Requirements
 
-Will be added before transition to beta.
+<!--
```

Comment:
>> Not required for Beta - but just pointing now that I would really like to see a way for monitoring if this works correctly for not. You should start thinking about that now.
>>
>> Same for troubleshooting...

> We are targeting alpha with phase 1, not beta. Do you mean to fill these sections before we target Beta? Or now for Alpha?

Sorry for not being clear.
What I meant is:
- I won't block the approval on this section not being filled int (as it's technically not required for Alpha)
- That said, I was trying to point out that this will be a potentially contentious topic during Beta approaval - I don't know how to monitor this feature nicely and we will need some good way for monitoring that for Beta approval. 
So I recommend you to start thinking about this now (and potentially already open another PR with some early proposals around that to kick the discussion).

Does that make sense?

Reply:
Yes, perfect. 

I added this to my TODO of things for Beta. Will open a PR for this with due time :)